controllers/auth.controller.js

```diff
@@ -91,6 +92,7 @@ module.exports.signup = (req, res) => {
 
                     user.save()
                         .then((user) => {
+                            Logger.addLog('authentication',`${req.userData.firstname} ${req.userData.lastname} 'created an account'`);
```

Comment:
This breaks the signup page. Not really sure why.
![image](https://user-images.githubusercontent.com/20412427/193482464-b67684ff-e1e9-471b-a2d5-5f46c215214f.png)